### PR TITLE
feat: add-fathom

### LINKED ISSUE #132: Add Fathom tracking
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Add privacy-focused analytics using https://usefathom.com/

### PR DESCRIPTION
**Pull request by agraebe**
Adding Fathom tracking to close #132

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -24,6 +24,17 @@ class MyApp extends App<MyAppProps & ReduxWrapperAppProps<RootState>> {
...
+    const tracker = window.document.createElement('script');
+    const firstScript = window.document.getElementsByTagName('script')[0];
+    tracker.defer = true;
+    tracker.setAttribute('site', `MVHUUREV`);
+    tracker.setAttribute('spa', 'auto');
+    tracker.src = 'https://cdn.usefathom.com/script.js';
+
+    firstScript.parentNode!.insertBefore(tracker, firstScript);
+  }
+
```
Comment (aulneau) [inline src/pages/_app.tsx:37]: I'd suggest we put this into this file: [app-init.tsx](https://github.com/blockstack/explorer/blob/feat/contract-call-debug/src/components/app-init.tsx), in a useEffect() wrapper.

Comment (aulneau) [reply src/pages/_app.tsx:37]: You'll also need to add in `typeof window !== 'undefined'` checks

Comment (aulneau) [reply src/pages/_app.tsx:37]: one more: please make it so this only runs in production

Comment (agraebe) [reply src/pages/_app.tsx:37]: 👍

**Comment by agraebe** [comment]
Yes, seems to work:

Screen Shot 2020-05-16 at 4 05 53 PM

**Pull request merged**